.github/workflows/wheels.yml

```diff
@@ -0,0 +1,34 @@
+name: Wheels
+on:
+  push:
+    branches:
+      - main
+  pull_request
+
+jobs:
+
+  build-wheel:
+    runs-on: ubuntu-18.04
```

Comment:
Do you have guidance for how to do builds on Mac or Windows?

Reply:
Let me extend the matrix to Mac, not sure about Windows though

Reply:
Done